README.md

```diff
@@ -40,7 +48,8 @@ you can run Continuous Clearing Tool as container or as a dotnet package,
  
  Execute them in the following order to achieve the complete License clearing process.
 
-1. **Package Identifier** - This executable takes `package-lock.json` or a `cycloneDX BOM` as input and provides a CycloneDX BOM file as output. For each of the component the availability in jfrog artifactory is identified and added in the BOM file.
+1. **Package Identifier** - This executable takes `package-lock.json` or a `cycloneDX BOM` as input and provides a SBOM file as output. For each of the component the dependency classification (development,internal) and the availability in jfrog artifactory is identified and added in the SBOM file.
```

Comment:
Why we're specifically mentioned as '`package-lock.json`'?
Instead can we mention Source files or Package files .